src/components/Editor/HighlightLine.js

```diff
@@ -46,60 +46,32 @@ function isDocumentReady(selectedSource, selectedLocation) {
 }
 
 export class HighlightLine extends Component<Props> {
-  isStepping: boolean = false;
```

Comment:
I found a way to use `props.pauseCommand` in one location rather than having to set and track this value.